packages/js/product-editor/src/components/woo-product-section-item/woo-product-section-item.tsx

```diff
@@ -15,6 +15,7 @@ type WooProductSectionItemProps = {
 	id: string;
 	tabs: ProductFillLocationType[];
 	pluginId: string;
+	children: ReactNode;
```

Comment:
```suggestion
	children: React.ReactNode;
```
Given thats what we do in the rest of the doc instead of importing the type explicitly.